.github/workflows/update-pull-request.yml

```diff
@@ -199,26 +189,28 @@ jobs:
     steps:
       - name: Checkout repository
```

Comment:
I don't fully follow the logic here, we are running checkout, then `gh pr checkout` and then the MetaMask checkout action too?

Reply:
`actions/checkout` to allow checking out the PR, then `MetaMask/action-checkout-and-setup` to install dependencies and handle caching. It skips the checkout step if the files are already there.